content/millennium/r4/devices/device.md

```diff
@@ -0,0 +1,109 @@
+* [Distinct identifier](https://hl7.org/fhir/r4/device-definitions.html#Device.distinctIdentifier){:target="_blank"}
+* [Manufacturer](https://hl7.org/fhir/r4/device-definitions.html#Device.manufacturer){:target="_blank"}
+* [Manufacture date](https://hl7.org/fhir/r4/device-definitions.html#Device.manufactureDate){:target="_blank"}
+* [Expiration date](https://hl7.org/fhir/r4/device-definitions.html#Device.expirationDate){:target="_blank"}
+* [Lot number](https://hl7.org/fhir/r4/device-definitions.html#Device.lotNumber){:target="_blank"}
+* [Serial number](https://hl7.org/fhir/r4/device-definitions.html#Device.serialNumber){:target="_blank"}
+* [Device name](https://hl7.org/fhir/r4/device-definitions.html#Device.deviceName){:target="_blank"}
+    * [Name](https://hl7.org/fhir/r4/device-definitions.html#Device.deviceName.name){:target="_blank"}
+    * [Type](https://hl7.org/fhir/r4/device-definitions.html#Device.deviceName.type){:target="_blank"}
+* [Model number](https://hl7.org/fhir/r4/device-definitions.html#Device.modelNumber){:target="_blank"}
+* [Type](https://hl7.org/fhir/r4/device-definitions.html#Device.type){:target="_blank"}
+* [Patient](https://hl7.org/fhir/r4/device-definitions.html#Device.patient){:target="_blank"}
+* [Owner](https://hl7.org/fhir/r4/device-definitions.html#Device.owner){:target="_blank"}
+
+## Terminology Bindings
+
+<%= terminology_table(:device, :r4) %>
+
+## Search
+
+Search for Devices that meet supplied query parameters:
+
+    GET /Device?:parameters
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true)%>
+
+### Parameters
+
+ Name       | Required?          | Type          | Description
+------------|--------------------|---------------|-------------------------------------------------------------------------------------------------------
+ `_id`      | This or `patient`  | [`token`]     | The logical resource id associated with the Device. Example: `2226920`
+ `patient`  | This or `_id`      | [`reference`] | The patient to whom the device is affixed. Example: `12345`
+ 
+### Headers
+
+<%= headers fhir_json: true %>
+
+### Example
+
+#### Request
+
+    GET https://fhir-ehr.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Device?patient=4478007
+    
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:R4_DEVICE_BUNDLE) %>
+
+### Errors
+
+The common [errors] and [OperationOutcomes] may be returned.
+
+## Retrieve by id
+
+List an individual Device by its id:
+
+    GET /Device/:id
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true)%>
+
+### Headers
+
+<%= headers fhir_json: true %>
+
+### Example
+
+#### Request
+
+    GET  https://fhir-ehr.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Device/15577765
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:R4_DEVICE) %>
+
+### Errors
+
+The common [errors] and [OperationOutcomes] may be returned.
+
```

Comment:
nit: extra line

Reply:
Removed it [here](https://github.com/cerner/fhir.cerner.com/pull/286/commits/4f208c84cb5b9f4f6973dd9983f9d4504d8d001a#diff-04bbe83c35d369961fb0c7e0ec9da3b1L105)

Reply:
+1